ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp

```diff
@@ -1957,11 +1972,11 @@ void py_module(py::module& module) {
         ttnn::selu,
         "scale", "Scale value", 1.0507,
         "alpha", "Alpha value", 1.67326);
-    detail::bind_unary_composite_floats(
+    detail::bind_unary_composite_threshold(
         module,
         ttnn::threshold,
         "threshold", "Threshold value",
-        "value", "Value value",
+        "value", "Replacing value",
```

Comment:
value to replace with

Reply:
That would look same as PyTorch, and hence i thought this would work